package.json

```diff
@@ -2,18 +2,25 @@
   "name": "ilp-protocol-stream",
   "version": "2.0.2",
   "description": "Interledger Transport Protocol for sending multiple streams of money and data over ILP.",
-  "main": "src/index.js",
+  "main": "dist/src/index.js",
+  "browser": {
+    "./dist/src/util/crypto_node.js": "./dist/src/util/crypto_browser.js",
+    "./src/util/crypto_node.ts": "./src/util/crypto_browser.ts",
+    "./dist/src/pool.js": false,
+    "./src/pool.ts": false
```

Comment:
Pool is unneeded because it's for running a connection server right, which you will never do in the browser ?

Reply:
Correct. Excluding it there isn't strictly required, but there's no reason to bundle the extra code unnecessarily.